Mobile Buy SDK/Mobile Buy SDK Tests/BUYRequestOperationTests.m

```diff
@@ -368,7 +368,7 @@ - (void)createExpectationDelay:(NSTimeInterval)delay block:(BOOL)block
 	}];
 	
 	if (block) {
-		[self waitForExpectationsWithTimeout:delay + 0.1 handler:^(NSError *error) {}];
+		[self waitForExpectationsWithTimeout:delay + 0.1 handler:[^(NSError *error) {} copy]];
```

Comment:
I imagine the the `copy` is in hope of eliminating the Buildkite crash? I'm pretty sure this accepts `nil` for the the `handler` parameter as well.